### ISSUE TITLE
provide single entry CLI mrQA (and "enable" shell completion)

### ISSUE THREAD

**Issue by yarikoptic** [Open-Minds-Lab/mrQA#23]
ATM functionality exposed on command line via a collection of scripts

```
        'console_scripts': [
            'protocol_compliance=mrQA.cli:main',
            'mr_proto_compl=mrQA.cli:main',
            'mrpc_subset=mrQA.run_subset:main'
        ],
```
which have no common prefix or consistency in naming. Makes it hard to impossible to recall what command I should run for doing `mrQA`.

Similarly to many other tools (`git`, `datalad`, ...) I recommend you to provide a single point of entry `mrqa` command line tool which would expose those three commands (may be renamed also to become a bit more descriptive).

Creating such interfaces is very easy with `click` library which we use e.g. for  dandi-cli. Here is click documentation on creating such "groups" of CLI: https://click.palletsprojects.com/en/8.1.x/commands/ . See e.g a little "advanced" example to define `dandi upload` where we also reuse definitions for some options (`--instance`) https://github.com/dandi/dandi-cli/blob/master/dandi/cli/cmd_upload.py etc. 

You could then also benefit from getting shell completion for the script - see https://click.palletsprojects.com/en/8.1.x/shell-completion/#shell-completion and our "helper" to make it easier to activate it: https://github.com/dandi/dandi-cli/blob/master/dandi/cli/cmd_shell_completion.py#L29 .

**Comment by raamana** [comment]
good point Yarik - we haven't gone that far in carefully organizing it but this a good time to clean this up before broader annoucement! prefixing all CLI scripts with `mrQA_` is a good idea.. we might have to update some docs cc @sinhaharsh

**Comment by raamana** [comment]
*all lowercase is intentional to save some effort with the shift key :)

**Comment by yarikoptic** [comment]
> prefixing all CLI scripts with `mrqa_`* is a good idea..

- so you don't want to just have a single `mrqa` entry script "to rule them all"?

**Comment by raamana** [comment]
i was referring to any future sub-functionalities we offer. one feature we haven't documented yet, which the `mrpc_subset` is the entry point, is the massively parallel processing capabilities of `mrQA` that we had to develop to process the ABCD dataset quickly. Perhaps we should call that `mrqa_parallel` and the rest just `mrqa` :)

**Comment by yarikoptic** [comment]
may be, or `mrqa parallel` ;-)

**Comment by raamana** [comment]
certainly! i am not sure how much more work that is compared to the easier of the options to add another entry point?

i can use that syntax being useful as mrQA starts to offer its full functionality in terms of both vertical audit (not implemented yet) and horizontal audits etc

```
mrqa horizontal --path /proj --style dicom
mrqa vertical   --path /proj --style bids
```

but its just work someone needs to do, and we can't prioritize this until we get vertical audit implemented and evaluate it on a few big/open datasets :)

**Comment by raamana** [comment]
also, that makes think @sinhaharsh , we should rename `--style` to be `--format` unless you have reasons you told me about and I forgot? :)
